### PR TITLE
Fix issue #895

### PR DESCRIPTION
**Pull request by Jorge-C**
Shows study ID in tables and study description

### CODE REVIEW

**Comment by Jorge-C** [comment]
The failure is stochastic.

**Comment by ElDeveloper** [comment]
:+1:

**Comment by ElDeveloper** [comment]
@squirrelo or someone else, can you take a quick pass at this PR, it looks good and ready to be merged.

**Comment by squirrelo** [comment]
Also need to add the study_id to the public studies page.

File: qiita_pet/templates/private_studies.html [hunk 1]
```diff
@@ -88,6 +88,7 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
         <thead>
             <tr>
                 <th>Title</th>
+                <th>Study ID</th>
```
Comment (squirrelo) [inline qiita_pet/templates/private_studies.html]: Also need to add this code to the user studies table. This only adds it to the shared studies table.

**Comment by squirrelo** [comment]
Couple of comments

**Comment by josenavas** [comment]
@Jorge-C merge conflict in here + comments from @squirrelo

**Comment by Jorge-C** [comment]
Thanks, I've solved the merge conflicts and @squirrelo's comment. Not sure what's going on with the test failures.

**Comment by antgonza** [comment]
Restarted the tests ...

**Comment by Jorge-C** [comment]
Thanks @antgonza, now ready.

**Pull request merged**